Automattic/jetpack
# Contact Form: Show form-specific blocks on top of the block library

Fixes #26889

#### Changes proposed in this Pull Request:

* Show form-specific blocks on top of the block library when the Form block is selected
* Filter the Core block list to remove less relevant blocks (Details on [#26889](https://github.com/Automattic/jetpack/issues/26889))
* Rename the forms block category from 'Grow' to 'Forms'

#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Checkout this branch and insert a Form on a Page/Post
* Check if the changes described above work as expected
